website/management/commands/update_faiss.py

```diff
@@ -0,0 +1,63 @@
+import os
+from pathlib import Path
+
+from django.core.management.base import BaseCommand
+
+from website.bot import embed_documents_and_save, is_api_key_valid, load_document, split_document
+
+
+class Command(BaseCommand):
+    help = "Update the FAISS database with new documents"
+
+    def handle(self, *args, **kwargs):
+        check_api = is_api_key_valid(os.getenv("OPENAI_API_KEY"))
+        if not check_api:
+            return self.stdout.write(self.style.ERROR(check_api))
```

Comment:
Why do you return `self.stdout.write` operation result?

Reply:
basically I used it for debugging . for notice the error clearly